src/main/kotlin/com/example/olaf/blogtutorial/HtmlController.kt

```diff
@@ -9,10 +9,12 @@ import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.server.ResponseStatusException
 
 @Controller
-class HtmlController(private val repository: ArticleRepository) {
+class HtmlController(private val repository: ArticleRepository,
+                    private val properties: BlogProperties) {
```

Comment:
```suggestion
class HtmlController(
    private val repository: ArticleRepository,
    private val properties: BlogProperties
) {
```
이렇게 써야 ktlint 가 좋아해요
지금 적용하신 indent 로는 ktlint 가 머지할 수 없게 할 것이에요 😂

Reply:
아 lint는 저것도 맞춰야하는군요 수정하겠습니다!